mir_eval/multipitch.py

```diff
@@ -0,0 +1,383 @@
+'''
+The goal of multiple f0 (multipitch) estimation and tracking is to identify
+all of the active fundamental frequencies in each time frame in a complex music
+signal.
+
+Conventions
+-----------
+Multipitch estimates are represented by a time series, and a corresponding list
```

Comment:
So to me, a "time series" means any sequence of temporal data, not the times at which that data occurs.  I would call the times at which some sequential temporal data occurs the "timebase".  Am I alone here?